pages/docs/tutorials/native/targeting-multiple-platforms.md

```diff
@@ -0,0 +1,135 @@
+---
+type: tutorial
+layout: tutorial
+title:  "Targeting Multiple Platforms"
+description: "Compiling with Kotlin/Native for different platforms"
+authors: Eugene Petrenko
+date: 2018-07-10
+showAuthorInfo: false
+issue: EVAN-5121
+---
+
+With Kotlin/Native can compile a native binary for 
```

Comment:
"With Kotlin/Native can compile a native binary for" 

either of these works:
With Kotlin/Native, you can compile a native binary for 
With Kotlin/Native, it is possible to compile a native binary for